Criipto.Signatures.IntegrationTests/CloseSignatureOrderTests.cs

```diff
@@ -0,0 +1,53 @@
+using Xunit;
+using Criipto.Signatures;
+using System.Collections.Generic;
+using System.IO;
+namespace Criipto.Signatures.IntegrationTests;
+
+public class CloseSignatureOrderTests
+{
+    public static string CLIENT_ID = System.Environment.GetEnvironmentVariable("CRIIPTO_SIGNATURES_CLIENT_ID")!;

```

Comment:
Seems to be the 3rd occurrence of this static setup, suggest to move to a general-purpose helper DSL instead (and also just have 1 test for existence of the env values)